apps/financial-aid/web-osk/src/utils/hooks/useUpdateApplication.ts

```diff
@@ -38,6 +41,7 @@ const useUpdateApplication = () => {
             event: ApplicationEventType.FILEUPLOAD,
             spouseEmail: form.emailAddress,
             spousePhoneNumber: form.phoneNumber,
+            spouseName: user?.name,
```

Comment:
When we create application shouldn't we use the `data` from national registry and null coalise it? Then we have this sometime before the spouse uploads data